release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1145,6 +1145,51 @@ With this update, the Ingress Operator no longer adds the `clientca-configmap` f
 [id="ocp-4-15-openshift-cli-bug-fixes"]
 ==== OpenShift CLI (oc)
 
+[discrete]
+[id="ocp-4-15-openshift-oc-mirror-bug-fixes"]
+==== oc-mirror
+
+Prior to this release, to filter operator packages by channel, for example, `mirror.operators.catalog.packages.channels`, you had to specify the default channel for the package, even if you did not intend to use the packages from that channel. Based on this information, the resulting catalog is considered invalid if the `imageSetConfig` does not contain the default channel for the package.
+
+[WARNING]
+====
+Modifying the default channel should not be taken lightly as it could have an impact on the Operator's upgrade paths. The use of `defaultChannel` is the responsibility of the end user.
+====
+
+This update introduces the `defaultChannel` field in the `mirror.operators.catalog.packages` section. You can now select a default channel. This action enables `oc-mirror` to build a new catalog that defines the selected channel in the `defaultChannel` field as the default for the package. 
+
+If you are interested in a specific channel, such as `stable-4.14`, and not the default channel, you can add the required default channel to the `imageSetConfig` under the `defaultChannel` of the package. This action enables `oc-mirror` to build a new catalog with the defined default channel and override the previous default channel, for example, in a catalog containing `operator A` with multiple channels and `stable-4.15` as the default channel.
```

Comment:
`This action enables oc-mirror to build a new catalog with the defined default channel and override the previous default channel`
This is repetitive. Prev sentence has the same.

Reply:
Let me remove it.